src/_components/ForNPOs/ProjectTimeline.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "https://esm.sh/react";
+
+export default function ProjectTimeline() {
+  return (
+    <section className="grow min-h-[550px] bg-neg text-black lg:px-20 py-16">
+      <div className="">
+        <h1 className=" font-bold text-3xl pl-12 pb-8">Our Project Timeline</h1>
+      </div>
+      <div className="grid lg:grid-cols-3">
+        <div className="px-12 py-8">
+          <h2 className="pb-4 text-xl text-primary font-bold">
+            Phase 1: Introductions
+          </h2>
+          <p>
+            In the initial stages of our project timeline, we'd like to meet and
+            discuss your organization's mission, the project idea, and what we
+            can do to help. We ask that organizations provide a proposal that
+            details the requested project. Not everything has to be fleshed out,
+            since we are open to collaborative scoping, but we would like as
+            much detail as possible so we can decide whether or not this project
+            is feasible for our team.
+          </p>
+        </div>
+        <div className="px-12 py-8 border-x-[1px] border-black">
```

Comment:
should be `lg:border-x-[1px]` so it doesn't show the border on the smaller breakpoint

Hard to see but that's the same border
<img width="739" alt="Screenshot 2024-11-20 at 6 22 22 PM" src="https://github.com/user-attachments/assets/f81890da-e412-41ea-8ac5-a66848d2a009">